openarm_hardware/include/openarm_hardware/motor.hpp

```diff
@@ -64,31 +103,46 @@ class Motor {
     int getStateTmos() const;
     int getStateTrotor() const;
     double getGoalPosition() const;
+    double getGoalVelocity() const;
     double getGoalTau() const;
 
     void setGoalPosition(double pos);
+    void setGoalVelocity(double vel);
     void setGoalTau(double tau);
     void setStateTmos(int tmos);
     void setStateTrotor(int trotor);
 private:
     double Pd, Vd;
-    double goal_position, goal_tau;
+    double goal_position,goal_velocity, goal_tau;
```

Comment:
FYI: We can detect format errors automatically by introducing linter/formatter:

```suggestion
    double goal_position, goal_velocity, goal_tau;
```